### PR TITLE
fix compile error in clang17

### PR DESCRIPTION
**Pull request by inabao**
fixed: #116

### CODE REVIEW

**Review by LHT129 — APPROVED** [review]
LGTM

**Review by wxyucs — APPROVED** [review]
lgtm

File: src/data_cell/graph_datacell_test.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@ TEST_CASE("graph basic test", "[ut][graph_datacell]") {
     auto max_degrees = {5, 12, 24, 32, 64, 128};
     auto max_capacities = {1, 100, 10000, 10'000'000, 32'179'837};
     std::vector<JsonType> graph_params;
-    std::string graph_param_temp = R"(
+    constexpr const char* graph_param_temp = R"(
```
Comment (ZhitaoShen) [inline src/data_cell/graph_datacell_test.cpp:45]: what is the warning here?

**Review by ZhitaoShen — APPROVED** [review]
LGTM

File: src/data_cell/graph_datacell_test.cpp (hunk 1 above)
Comment (inabao) [reply src/data_cell/graph_datacell_test.cpp:45]: I have pasted the screenshot in issue #128

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**